alcs-frontend/src/app/shared/card/card.component.scss

```diff
@@ -18,18 +18,33 @@
   align-content: center;
 }
 
+.lower-card > div {
+  display: flex;
+  align-items: center;
+}
+
 .assignee {
   border-radius: 50%;
   padding: 6px;
   border: 1px solid #0a5a9c;
 }
 
+.activeDays {
```

Comment:
classes usually use '-' not camelCase  => active-days

Reply:
Good catch! I even used - in the same file >.>
Fixed!